### ISSUE TITLE
Launching Parity with a Ledger plugged in causes Parity to become unresponsive.

### ISSUE THREAD

**Issue by MicahZoltu** [paritytech/parity#5672]
1. Plug in a ledger device (e.g., Ledger Nano S)
2. Launch Parity.
3. Launch Parity UI.
4. --> Notice that the UI never becomes usable.

**Comment by 5chdn** [comment]
Could you sate OS and Parity version? Thanks

**Comment by nikiwiles** [comment]
For me it was Windows 10, with Parity 1.6.6.

**Comment by MicahZoltu** [comment]
Windows 10
Parity//v1.6.6-beta-8c6e3f3-20170411/x86_64-windows-msvc/rustc1.16.0

**Comment by mpolavieja** [comment]
The same for me.   Windows 7 Home and Parity//v1.6.7-beta-e128418-20170518/x86_64-windows-msvc/rustc1.17.0

Quite annoyed for several hours until I unplugged Ledger Nano, killed parity process and launch parity again.  Then parity worked again.    To make sure I shutted down parity and plug in the Ledger Nano again, and Parity didn´t launch.

**Comment by JanKalin** [comment]
Same here. Windows 7 Pro, Parity/v1.6.7-beta-e128418-20170518/x86_64-windows-msvc/rustc1.17.0

Perhaps it's the Ledger firmware - I haven't yet upgraded to 1.3.1

**Comment by arkpar** [comment]
Can't reproduce this.
@MicahZoltu did you start ethereum app on the device before starting parity? What's your ledger firmware version and ethereum app version? Does it still happen if you update to latest versions?
Could you start parity with `-l hardware_wallet::ledger=trace` reproduce the issue and post logs?

**Comment by MicahZoltu** [comment]
Hmm, I'm not sure how to reproduce it.  I just tried launching Parity with my Nano plugged in both with the Ethereum app opened and closed and with it in browser mode and not browser mode.

I am currently on Ledger 1.3.1, I'm not sure if I upgraded at some point since I ran into this issue.

**Comment by MicahZoltu** [comment]
Perhaps @nikiwiles, @mpolavieja, or @JanKalin can still reproduce the issue?

**Comment by debris** [comment]
@tomusdrw @folsen iirc you have fixed that, am I right? Can we close this?

**Comment by JanKalin** [comment]
I can confirm that with the Parity//v1.7.6-stable-1953533-20171013/x86_64-windows-msvc/rustc1.20.0
, firmware 1.3.1 and app version 1.0.18 the problem is no longer there - I can use Ledger in Parity without problems.

**Comment by tomusdrw** [comment]
Yup, the blocking issue should be solved. @JanKalin I've opened a separate issue to track the crash: #6805
